frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx

```diff
@@ -56,11 +70,20 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs }) => {
                         )}
                       </StackItem>
                       <StackItem className="co-truncate co-nowrap">
-                        <span className="co-resource-item">
+                        <span className="co-resource-item odc-gitops-details__env-section__co-resource-item">
                           <ResourceIcon kind="Project" />
                           <span className="co-resource-item__resource-name">{env.environment}</span>
                         </span>
                       </StackItem>
+                      <StackItem className="co-truncate co-nowrap">
+                        {env.environment && (
```

Comment:
Add a check for `argocdLink` here. 
```suggestion
                        {env.environment && argocdLink && (
```